.github/workflows/test_installer.yml

```diff
@@ -0,0 +1,39 @@
+name: Install Local Version of Opik
+
+on:
+    workflow_dispatch:
+
+jobs:
+    test_installation:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout repo
+              uses: actions/checkout@v3
+              with:
+                ref: ${{ github.ref }}
+            
+            - name: Setup Python
+              uses: actions/setup-python@v5
+              with:
+                python-version: 3.12
+
+            - name: Install Opik
+              run: pip install sdks/python
+
+            - name: Install Test Dependencies
+              run: |
+                pip install -r tests_end_to_end/test_requirements.txt
+                playwright install
+
+            - name: Install Opik
+              run: |
+                cd deployment/docker-compose
+                docker compose up --detach
+
+            - name: Wait
+              run: sleep 5
+
+            - name: Check app is up via the UI
+              run: |
+                pytest -v -s tests_end_to_end/installer/test_app_status.py
```

Comment:
Most likely, you want a generic command to run all existing tests in the installer folder.

Reply:
Will do if there is a need to actually add more tests in the installer folder. currently planning on only having some basic tests to check that the UI is actually visible, there is most likely no need to spread this over multiple files/suites. More specific tests involving the UI will live in tests_end_to_end/experiments, tests_end_to_end/datasets or whatever other file structure we decide on

Reply:
you shoud add stopping docker compose, like 
```
- name: Stop opik server
          if: always()
          run: |
            cd ../../deployment/docker-compose
            docker compose down
            cd -
            ```

Reply:
added